README.md

```diff
@@ -134,6 +134,33 @@ person2:
     age: 20
 ```
 
+If the data is a list of items without a key at the top level:
```

Comment:
Thanks for helping our docs. I think this may be better in the `examples` section.